numpy/ma/core.pyi

```diff
@@ -18,13 +18,15 @@ from typing_extensions import TypeIs, TypeVar
 import numpy as np
 from numpy import (
     _AnyShapeT,
+    _ArrayInt_co,
```

Comment:
This `NDArray[integer | np.bool]` is simple enough to inline here, given that it's only used twice (in `__init__.pyi` that's 30x). Importing private type-aliases like these can be risky, as it's not obvious that you should check for references to some private type alias if you rename or remove it (I believe I recently introduced a bug like this, which you managed to track down if I remember correctly?).